packages/web/src/pages/sign-up-page/utils/determineAllowedRoutes.ts

```diff
@@ -16,7 +16,7 @@ export const determineAllowedRoute = (
 } => {
   const attemptedPath = requestedRoute.replace('/signup/', '')
   // Have to type as string[] to avoid too narrow of a type for comparing against
-  let allowedRoutes: string[] = [SignUpPath.createEmail] // create email is available by default
+  let allowedRoutes: string[] = [SIGN_UP_PAGE, SignUpPath.createEmail] // create email is available by default
```

Comment:
What was the motivation for this? I originally left it out so that it would append the `/create-email` to the `/signup`. 
I think with this change the `/create-email` path would be pretty much just a dead path. Which is fine if we want to get rid of it, but then we should remove `SignUpPath.createEmail` as well

Reply:
so im really confused about this path stuff, cause when you go directly to /signup, we get some weird routing glitch where it redirects to /create-email, but the ui doest... this was an attempt to try and make things work. would love your advice on this, cause im seeing bugs with landing on /signup, and navigating backwards...

Reply:
oh i see your second comment, yes im quite confused what is going on with why going back isnt working in this pr...